py_zipkin/zipkin.py

```diff
@@ -143,6 +154,20 @@ def __init__(
         if self.sample_rate is not None and not (0.0 <= self.sample_rate <= 100.0):
             raise ZipkinError('Sample rate must be between 0.0 and 100.0')
 
+        if not set(include).issubset(set(STANDARD_ANNOTATIONS.keys())):
+            raise ZipkinError(
+                'Only %s are supported as annotations' %
+                STANDARD_ANNOTATIONS.keys()
+            )
+        else:
+            # get a list of all of the mapped annotations
+            self.annotation_filter = {
```

Comment:
I kinda don't like this implementation on principle, because it's O(number of types of standard annotations) rather than O(number of included annotation types). But this is probably a silly reason to dislike it, since I really doubt we'll ever end up with more than a tiny amount of standard annotations.